### PR TITLE
DEVX-2714: update Confluent CLI syntax and checking ids

### PR DESCRIPTION
**Pull request by ybyzek**
### Description 

https://confluentinc.atlassian.net/browse/DEVX-2714

_What behavior does this PR change, and why?_

- update CLI change for ksql cluster
- Better to use json with jq over grep

### Author Validation

_Describe the validation already done, or needs to be done, by the PR submitter._

### Reviewer Tasks

_Describe the tasks/validation that the PR submitter is requesting to be done by the reviewer._

### CODE REVIEW

File: cloud-etl/create_ksqldb_app.sh [hunk 1]
```diff
@@ -27,8 +27,8 @@ ccloud::validate_credentials_ksqldb "$KSQLDB_ENDPOINT" "$CONFIG_FILE" "$KSQLDB_B
 echo -e "Create output topics $KAFKA_TOPIC_NAME_OUT1 and $KAFKA_TOPIC_NAME_OUT2, and ACLs to allow the ksqlDB application to run\n"
 confluent kafka topic create $KAFKA_TOPIC_NAME_OUT1
 confluent kafka topic create $KAFKA_TOPIC_NAME_OUT2
-ksqlDBAppId=$(confluent ksql app list | grep "$KSQLDB_ENDPOINT" | awk '{print $1}')
-confluent ksql app configure-acls $ksqlDBAppId $KAFKA_TOPIC_NAME_IN $KAFKA_TOPIC_NAME_OUT1 $KAFKA_TOPIC_NAME_OUT2
+ksqlDBAppId=$(confluent ksql cluster list | grep "$KSQLDB_ENDPOINT" | awk '{print $1}')
```
Comment (rspurgeon) [inline cloud-etl/create_ksqldb_app.sh]: Would it make sense to convert this to using `json` output as well?

Comment (ybyzek) [reply cloud-etl/create_ksqldb_app.sh]: @rspurgeon good catch.  Updated in latest commit

File: cp-quickstart/start-cloud.sh [hunk 2]
```diff
@@ -93,13 +93,13 @@ retry $MAX_WAIT ccloud::validate_ccloud_ksqldb_endpoint_ready $KSQLDB_ENDPOINT |
 print_pass "Confluent Cloud KSQL is UP"
 
 printf "Obtaining the ksqlDB App Id\n"
-CMD="confluent ksql app list -o json | jq -r '.[].id'"
+CMD="confluent ksql cluster list -o json | jq -r '.[].id'"
```
Comment (rspurgeon) [inline cp-quickstart/start-cloud.sh:96]: FYC: Now that we're using json, we can be more reliable with these commands. The array might contain more than one ksqldb app, and then we are storing the wrong app id and maybe causing issues for the user.  Instead we could be specific with `jq`...

```
confluent ksql cluster list -o json | jq --arg KSQLDB_ENDPOINT "$KSQLDB_ENDPOINT" -r '.[] | select(.endpoint==$KSQLDB_ENDPOINT) | .id'
```

Comment (ybyzek) [reply cp-quickstart/start-cloud.sh:96]: Point take.  In this case: if a user is running this script `cp-quickstart/start-cloud.sh`, it's a new environment so there is only one ksqlDB app.

Comment (rspurgeon) [reply cp-quickstart/start-cloud.sh:96]: Yes, unless the user is running other commands during long scripting runs (like while the app provisions) and then results in confused state in the CLI.

**Review by rspurgeon — APPROVED** [review]
lgtm with minor comments. Verified the beginner-cloud and cp-quickstart (didn't run etl).

**Comment by ybyzek** [comment]
Thanks for the review @rspurgeon

**Pull request merged**